charts/opentelemetry-kube-stack/values.yaml

```diff
@@ -332,6 +332,78 @@ defaultCRConfig:
+      includeCollectorLogs: true
+      # Enabling this writes checkpoints in /var/lib/otelcol/ host directory.
+      # Note this changes collector's user to root, so that it can write to host directory.
+      storeCheckpoints: false
+      # The maximum bytes size of the recombined field.
+      # Once the size exceeds the limit, all received entries of the source will be combined and flushed.
+      maxRecombineLogSize: 102400
+    # Configures the collector to collect host metrics.
+    # Adds the hostmetrics receiver to the metrics pipeline
+    # and adds the necessary volumes and volume mounts.
+    # Best used with mode = daemonset.
+    # See https://opentelemetry.io/docs/kubernetes/collector/components/#host-metrics-receiver for details on the receiver.
+    hostMetrics:
+      enabled: false
+    # Configures the Kubernetes Processor to add Kubernetes metadata.
+    # Adds the k8sattributes processor to all the pipelines
+    # and adds the necessary rules to ClusteRole.
+    # Best used with mode = daemonset.
+    # See https://opentelemetry.io/docs/kubernetes/collector/components/#kubernetes-attributes-processor for details on the receiver.
+    kubernetesAttributes:
+      enabled: false
+      # When enabled the processor will extra all labels for an associated pod and add them as resource attributes.
+      # The label's exact name will be the key.
+      extractAllPodLabels: false
+      # When enabled the processor will extra all annotations for an associated pod and add them as resource attributes.
+      # The annotation's exact name will be the key.
+      extractAllPodAnnotations: false
+    # Configures the collector to collect node, pod, and container metrics from the API server on a kubelet..
+    # Adds the kubeletstats receiver to the metrics pipeline
+    # and adds the necessary rules to ClusteRole.
+    # Best used with mode = daemonset.
+    # See https://opentelemetry.io/docs/kubernetes/collector/components/#kubeletstats-receiver for details on the receiver.
+    kubeletMetrics:
+      enabled: false
+    # Configures the collector to collect kubernetes events.
+    # Adds the k8sobject receiver to the logs pipeline
+    # and collects kubernetes events by default.
+    # Best used with mode = deployment or statefulset.
+    # See https://opentelemetry.io/docs/kubernetes/collector/components/#kubernetes-objects-receiver for details on the receiver.
+    kubernetesEvents:
+      enabled: false
+    # Configures the Kubernetes Cluster Receiver to collect cluster-level metrics.
+    # Adds the k8s_cluster receiver to the metrics pipeline
+    # and adds the necessary rules to ClusteRole.
+    # Best used with mode = deployment or statefulset.
+    # See https://opentelemetry.io/docs/kubernetes/collector/components/#kubernetes-cluster-receiver for details on the receiver.
+    clusterMetrics:
+      enabled: false
+    # Configures the batch processor for telemetry batching prior to sending
+    batchProcessor:
```

Comment:
I was thinking this would be a useful thing that most users would want to add in... I can remove some of the proxying and instead just set some default values for this chart

Reply:
setting the batchprocessor as a default processor feels better than a preset for it.  All the presets exist to handle both component and k8s setup, and the batch processor has no k8s setup.